frontend/packages/ux-editor/src/components/config/editModal/EditGrid/EditGrid.module.css

```diff
@@ -8,6 +8,7 @@
 }
 
 .tabs {
+  flex-wrap: wrap;
```

Comment:
Any reason this is added?

Reply:
I see that you have mentioned in the PR description, but I dont necessarily think the overflow on those tabs is an issue 🫣

Reply:
It's to prevent a horizontal scroll bar appearing. Do you think we should keep it scrollable instead?

Reply:
I think it was intentional the time it was implemented. But also personal preferences I guess! I do not have strong opinions on this tho😅

Reply:
Ok good point, i will do the safe bet and revert that change then 👍